vhdbuilder/packer/components.json

```diff
@@ -245,67 +226,50 @@
       "downloadURL": "mcr.microsoft.com/aks/ip-masq-agent-v2:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v0.1.9",
         "v0.1.10"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.26.9",
-        "v1.28.6",
-        "v1.29.3",
         "v1.29.4"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.26.11",
```

Comment:
removing azure disk & file driver 1.26.* is ok since they are only for aks 1.26, pls keep other images in vhd

Reply:
Will bring the following versions back:
"v1.28.9",
"v1.29.3",
"v1.29.4",
"v1.30.0",

Will ping you to see if we need the older patch versions (1.29.3 and 1.30.0)